src/components/EditMascot/EditMascot.js

```diff
@@ -99,19 +99,19 @@ export default function EditMascot() {
   const getMascotReward = (length) => {
     switch (length) {
       case 3: {
-        return "Q6AkAx4llp";
+        return "Ew7kanM6BV";
```

Comment:
would be nice at some point to give symbolic names to these constants... otherwise such code is impossible to review :)